### DISCUSSION TITLE
failed to create pg.tables: Invalid URL

### DISCUSSION THREAD

**Discussion by hawkiboy** [supabase/supabase#20984] [Questions]
Just opened account on supabase. First thing I try to do is create a table. The most basic table with just a couple of columns. I have tried various ways but always gett "failed to create pg.tables: Invalid URL".  I managed to get the sql editor to AI generate a table. Then I went in and tried to import a csv file. Everything looked great but it still hung up on that failed to create pg.tables: Invalid URL. As a newbie I can not figure out how this happens to me and I don't see a single comment about it anywhere. Could it have anything to do with my internet connection? I tried changing my VPN (Private internet access) to point to a few other locations,... I did get an import to work once. Not sure why. Any insights would be welcome

**Comment by GaryAustin1** [comment]
Try clearing your local storage and flushing your browser cache, or a different browser.
